recipes/torchquad/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "torchquad" %}
+{% set org = "esa" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "bca1a3612a31d32b9c73c80344cabdc1dbaa5aab3bfeb04f9c91cb57d0194b1c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip 
+    - setuptools
+  run:
+    - python >=3.8
+    - matplotlib >=3.3.3
```

Comment:
Unless you need a pyqt GUI, best to depend on `matplotlib-base`

Reply:
Changed it to matplotlib-base